ipatests/test_integration/test_trust.py

```diff
@@ -90,6 +92,49 @@ def remove_trust(self, ad):
         tasks.remove_trust_with_ad(self.master, ad.domain.name)
         tasks.clear_sssd_cache(self.master)
 
+    def check_filter_users(self, realm, aduser, provider=None, offline=False):
+        master = self.master
+        user1 = 'some_user@some.domain'
+        tasks.backup_file(self.master, paths.SSSD_CONF)
```

Comment:
It is better to use tasks.FileBackup for backup restore since tasks.restore_files restores all files that have been backuped during the test run, which can include the files modified in class setup (usually done in install()).

tasks.FileBackup  on the other hand preserves copy of exactly one file and also preserves selinux context.